samples/Calling/src/app/utils/AppUtils.ts

```diff
@@ -59,7 +60,7 @@ export const createRoomId = async (): Promise<string> => {
  * Joins an ACS room with a given roomId and role
  */
 // eslint-disable-next-line @typescript-eslint/no-explicit-any
-export const joinRoom = async (userId: string, roomId: string, role): Promise<void> => {
+export const joinRoom = async (userId: string, roomId: string, role: Role): Promise<void> => {
```

Comment:
do we need to conditional hide this as well?

Reply:
Yes when we merge it into main.